framework/components/ecl-forms/ecl-forms-form-groups/ecl-forms-form-groups.twig

```diff
@@ -1,27 +1,20 @@
-<form>
```

Comment:
Related to labels in forms, even demo such: the text input demo is not having the `ecl-form-label` and might be misleading for validating design team:

![label-text-input](https://user-images.githubusercontent.com/1923476/29030545-ab46ebaa-7b8b-11e7-9b44-0cbce21ee80c.png)